headers/Detour/ILCallback.hpp

```diff
@@ -21,7 +21,7 @@ namespace PLH {
 			}
 
 			// asm depends on this specific type
-			uint64_t m_arguments[];
+			uint64_t m_arguments[0];
```

Comment:
Please change this to m_arguments[1]. C99 has 'flexible array members' which was what i intended to use here, but i didn't realize that is not supported in any version of c++ yet. Unfortunately [0] is equally invalid. The best work around is to make it a valid array size but then use undefined behavior to read beyond it (which is fine because I alloc properly).

Reply:
Done. I re-ran the tests with the change and got the same results